hapi-utils/src/main/java/com/hederahashgraph/fee/FeeBuilder.java

```diff
@@ -162,11 +162,11 @@ public static FeeObject getFeeObject(FeeData feeData, FeeData feeMatrices, Excha
    *     <li>bytes string memo - get memo size from transaction</li>
    * </ul>
    */
-  public static int getCommonTransactionBodyBytes(TransactionBody txBody) throws InvalidTxBodyException {
+  public static long getCommonTransactionBodyBytes(TransactionBody txBody) throws InvalidTxBodyException {
     if (txBody == null) {
       throw new InvalidTxBodyException("Transaction Body not available for Fee Calculation");
     }
-    int memoSize = 0;
+    long memoSize = 0;
     if (txBody.getMemo() != null) {
       memoSize = txBody.getMemoBytes().size();
```

Comment:
`txBody.getMemoBytes().size()` will return an `int` so it's better to keep `int memoSize = 0;` and do a cast when we return for `getCommonTransactionBodyBytes`

Reply:
Fixed. Thanks!